util.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
```

Comment:
nit: In accordence with my new-years resolution to get rid of files and packages named "util" or "lib", I suggest we rename this file to "user.go" (and "user_test.go" for it's accompanying test file).